src/sass/partials/_form.scss

```diff
@@ -1,104 +1,7 @@
-form {
-  margin: 0 auto;
-  width: 100%;
-  max-width: 30em;
-  padding: $padding-margin-value;
-}
-
-input {
-  display: block;
-  width: 100%;
-  padding: $form-padding-value;
-  margin: $form-padding-value auto $form-padding-value auto;
-  border: none;
-  color: $black;
-  background-color: #FDEFF0;
-  font-weight: 100;
-  box-shadow: 1px 1px 1px 1px rgba(0,0,0,.1) inset
-}
-
-input::-webkit-input-placeholder, textarea::-webkit-input-placeholder {
-  color: #FB4553;
-}
-
-label {
-  font-weight: 400;
-}
-
-input[type=email] {
-  display: block;
-  padding: $form-padding-value;
-  margin: 0 auto $form-padding-value auto;
-  width: 100%;
-  border: none;
-  color: $black;
-  font-weight: 100;
-}
-
-input[type=number] {
-  display: block;
-  padding: $form-padding-value;
-  margin: 0 auto $form-padding-value auto;
-  width: 100%;
-  border: none;
-  color: $black;
-  font-weight: 100;
-}
-
-input[type=submit] {
-  display: block;
-  padding: $form-padding-value;
-  margin: 0 auto $form-padding-value auto;
-  width: 100%;
-  border: none;
-  opacity: 1;
-  background-color: $pop-pink;
-  font-weight: 100;
-  color: $white;
-  &:hover {
-    opacity: .8;
-  }
-}
-
-input[type=tel] {
-  display: block;
-  padding: $form-padding-value;
-  margin: 0 auto $form-padding-value auto;
-  width: 100%;
-  border: none;
-  color: $black;
-  font-weight: 100;
-}
-
-input[type=text] {
-  display: block;
-  padding: $form-padding-value;
-  margin: 0 auto $form-padding-value auto;
-  width: 100%;
-  border: none;
-  color: $black;
-  font-weight: 100;
-}
-
-input[type=url] {
-  display: block;
-  padding: $form-padding-value;
-  margin: 0 auto $form-padding-value auto;
-  width: 100%;
-  border: none;
-  color: $black;
-  font-weight: 100;
-}
-
-textarea {
-  display: block;
-  width: 100%;
-  padding: $form-padding-value;
-  margin: 0 auto $form-padding-value auto;
-  border: none;
-  color: $black;
-  background-color: #FDEFF0;
-  font-weight: 100;
-  box-shadow: 1px 1px 1px 1px rgba(0,0,0,.1) inset
-
-}
+span.help-text {
```

Comment:
Since we import bootsrap first thing any styles you write afterwards naturally override it.